libcudacxx/include/cuda/std/__cccl/compiler.h

```diff
@@ -100,4 +100,15 @@
 #  define _CCCL_CUDACC_BELOW_12_3
 #endif // defined(_CCCL_CUDACC) && _CCCL_CUDACC_VER < 1203000
 
+// Enable us to selectively silence host compiler warnings
+#define _CCCL_TOSTRING2(_STR) #_STR
+#define _CCCL_TOSTRING(_STR)  _CCCL_TOSTRING2(_STR)
```

Comment:
Lets name this with a _
```suggestion
#define _CCCL_TO_STRING2(_STR) #_STR
#define _CCCL_TO_STRING(_STR)  _CCCL_TO_STRING2(_STR)
```

Reply:
Updated on all appearances.